learn-more.html

```diff
@@ -2,12 +2,79 @@
+    <link href="l-m-style.css" rel="stylesheet"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lexend+Giga&display=swap" rel="stylesheet"/>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+ 
+  <body id="everything"> 
+    <div id="margin1">to delete</div>  <!--margins of the body-->
+    <div id="margin2">to delete</div>
+    <section id="header"><div id="title">Learn More</div><div id="take">or How To Have Something To Do in the Summer<hr><hr> </div></section>
+
+    <div id="post">  <!--I put all the content of the page into one div (minus the footer)-->
+    <div id="body">
+        <div id="main"></div>  
+        <div id="sidebar">
+          <a href="blog-post.html"><button id="date">July 30</button></a>
+          <h4>Just Another Post</h4>
+          <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis<hr>
+          <a href="blog-post.html"><button id="date">July 28</button></a>
+          <h4>And Another Post</h4>
+          <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis<hr>
+          <a href="blog-post.html"><button id="date">July 24</button></a>
+          <h4>One More Post</h4>
+          <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus<hr>
+          <h4>Something of Note</h4>
+          <img src="https://i.imgur.com/uxJLfFs.png" alt="Mount Crested Butte" class="side";/>
+           Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+          <a href="learn-more.html"><br><button id="more">Learn More</button></a><hr>
+          <h4>Something of Less Note</h4>
+          <img src="https://i.imgur.com/SMwTgQ5.png" alt="Black Canyon of the Gunnison" class="side";/>
+          Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+          <br><br>
+          <img src="https://i.imgur.com/bLtlF3Q.png" alt="Mountains and Trees in Alma, CO" class="side";/>
+          Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+          <a href="learn-more.html"><br><button id="more">Learn More</button></a>
+      </div>

```

Comment:
Here is another spot that we can use our padding/margin to create a bit of space between the two columns